pkg/packaging/helpers.go

```diff
@@ -33,9 +33,11 @@ func (p *PackageOptions) setOsqueryVersionInCtx(ctx context.Context) {
 		return
 	}
 
-	osquerydVersionTrimmed := strings.TrimPrefix(strings.TrimSpace(stdout), "osqueryd version ")
+	packagekit.SetInContext(ctx, packagekit.ContextOsqueryVersionKey, osqueryVersionFromVersionOutput(stdout))
+}
 
-	packagekit.SetInContext(ctx, packagekit.ContextOsqueryVersionKey, osquerydVersionTrimmed)
+func osqueryVersionFromVersionOutput(output string) string {
+	return strings.TrimPrefix(strings.TrimPrefix(strings.TrimSpace(output), "osqueryd.exe version "), "osqueryd version ")
```

Comment:
I wonder if there will be times osquery prints something else, and splitting on `version` would work better. 🤷

Reply:
Updated!